style.css

```diff
@@ -0,0 +1,90 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Nunito', sans-serif;
+  box-sizing: border-box;
+}
+
+#main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: beige;
+  margin: 20px;
+  padding: 50px;
+}
+
+#title {
+  font-weight: 600;
+  font-size: 2.5rem;
+  padding: 10px 0;
+}
+
+#main h6 {
+  font-weight: 400;
+  font-size: 1.3rem;
+  padding: 15px 0;
+}
+
+#img-div {
+  padding: 20px 10px;
+  background-color: white;
+}
+
+#image {
+  max-width: 100%;
+  height: auto;
+  display: block;
+}
+
+#img-caption {
+  text-align: center;
+  padding-top: 15px;
+}
+
+#tribute-info {
+  width: 65%;
+}
+
+#tribute-info h5 {
+  padding: 3rem;
+  font-size: 1.3rem;
+  font-weight: 500;
+  text-align: center;
+}
+
+#tribute-info ul li {
+  padding: 10px 0;
+}
+
+#tribute-info > ul > li > p {
+  line-height: 25px;
+  font-size: 1.1rem;
+  font-weight: 300;
+}
+
+#tribute-info blockquote {
+  padding: 30px 0 12px;
+  font-size: 1.2rem;
+  font-weight: 200;
+  font-style: italic;
+}
+
+.quote {
+  font-size: 1.2rem;
+  padding: 35px 0 10px 0;
+}
```

Comment:
- [ ] the Wikipedia link has a line under it. it is the default in HTML. can you add `link decoration: none` here to remove it.

Reply:
I suggest you elaborate more on why you requested this change, as it might not seem obvious for the reviwee. You can refer to this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) on how you can communicate better when doing reviews.